sjb/config/test_cases/test_branch_ansible_service_broker.yml

```diff
@@ -0,0 +1,93 @@
+---
+parent: 'common/test_cases/origin_built_release.yml'
+overrides:
+  sync:
+    - "openshift,ansible-service-broker=master"
+  junit_analysis: False
+  timer: 'H H * * *'
+  email:
+    - ernelson@redhat.com
+extensions:
+  actions:
+    - type: "script"
+      title: "determine the release commit for origin images"
+      repository: "origin"
+      script: |-
+        jobs_repo="/data/src/github.com/openshift/aos-cd-jobs/"
+        export ORIGIN_COMMIT=$(git log -1 --pretty=%h)
+        echo $ORIGIN_COMMIT > "${jobs_repo}/ORIGIN_COMMIT"
+        source hack/lib/init.sh
+        export ORIGIN_BUILT_VERSION=$(os::build::rpm::format_nvra)
+        echo $ORIGIN_BUILT_VERSION > "${jobs_repo}/ORIGIN_BUILT_VERSION"
+    - type: "script"
+      title: "build Origin"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/local/bin/linux/amd64 hack/env make build
+    - type: "script"
+      title: "build Service Catalog binary"
+      repository: "service-catalog"
+      timeout: 1800
+      script: |-
+        hack/env make build
+    - type: "script"
+      title: "build Service Catalog image"
+      repository: "service-catalog"
+      timeout: 1800
+      script: |-
+        hack/env hack/build-source-image.sh
+        hack/env make build-images
+    - type: "script"
+      title: "build origin images"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        hack/build-local-images.py
+    - type: "script"
+      title: "start cluster with Service Catalog"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        if [[ "${PULL_BASE_REF:-master}" == "release-3.9" ]]; then
+          ./_output/local/bin/linux/amd64/oc cluster up --loglevel=5 --version=latest --service-catalog
+        else
+          ./_output/local/bin/linux/amd64/oc cluster up --loglevel=5 --tag=latest --enable=service-catalog
+        fi
+        ./_output/local/bin/linux/amd64/oc login -u system:admin
+        ./_output/local/bin/linux/amd64/oc describe po --all-namespaces
```

Comment:
Everything up to here in the extensions section can go away with catalog set as the parent.